capstone/fitness/src/main/webapp/progress.html

```diff
@@ -0,0 +1,152 @@
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+  <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+  <script src="progress_disp.js"></script>
+</head>
+
+<div class="topnav">
+  <a class="active" href="/">Home</a>
+  <a href="/calendar.html">Calendar</a>
+  <a class="active" href="/progress.html">Progress</a>
+  <a href="/view-data.html">Data</a>
+</div>
+
+<body id="content">
+  <div class="row">
+    <div class="col-sm">
+      <div class="row">
+        <!-- Pagination -->
+        <div class="col-sm">
+          <!-- Sorting -->
+          <!--TODO(ijelue): Add Servlet for automatic sorting logic. -->  
+          <input type="radio" id="uncomplete" name="sorting" value="uncomplete" checked>
+          <label for="uncomplete">Show Uncomplete</label><br>
+          <input type="radio" id="all" name="sorting" value="all">
+          <label for="all">Show All</label><br>
+          <input type="radio" id="complete" name="sorting" value="complete">
+          <label for="complete">Show Complete</label><br>
+
+          <!-- Submit Session -->
+          <!--TODO(ijelue): Connect to existing submit session button on home page. -->
+          <button type="button" class="btn btn-primary btn-sm">Submit Session</button>  
+        </div>
+        <div class="col-sm">
+          <!-- Goal Step Count Per Page -->
+          <!--TODO(ijelue): Add Servlet for automatic number restriction logic. -->
+          <button class="btn btn-info btn-sm">Go!</button>
+          <input type="number" id="count" name="count" min="1" max="10">
+          <label for="count">Goal Steps Per Page</label><br>
+
+          <!-- Prev/Next Page Stuffs -->
+          <!--TODO(ijelue): Add Servlet for goal step traversal. -->
+          <ul class="pagination">
+            <li class="page-item"><a class="page-link" href="#">Previous</a></li>
+            <li class="page-item"><a class="page-link" href="#">1</a></li>
+            <li class="page-item"><a class="page-link" href="#">Next</a></li>
+          </ul>
+        </div>
+      </div><br>
+      <div class="row">
+        <!-- Goal Steps -->
+        <!--TODO(ijelue): Add Servlet and javascript for automatic generation of goal step buttons. -->
+        <div id="model">
+          <button type="button" class="btn btn-light" onclick="displayGoalStep()">Start</button><br>
+          <div id="button0" class="goal-desc"></div>
```

Comment:
Consider naming this something more descriptive.

Reply:
done